src/System.IO.FileSystem/src/System/IO/FileSystem.Win32.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.InteropServices;
+
+namespace System.IO
+{
+    internal static partial class FileSystem
+    {
+        public static void Encrypt(string path)
+        {
+            if (path == null)
+                throw new ArgumentNullException(nameof(path));
+
+            string fullPath = Path.GetFullPath(path);
+
+            if (!Interop.Kernel32.EncryptFile(fullPath))
+            {
+                int errorCode = Marshal.GetLastWin32Error();
```

Comment:
You can factor this error handling into a helper.

Reply:
done